share/man/man1/urlwatch.1

```diff
@@ -41,6 +41,9 @@ List telegram chats the bot is joined to
 .TP
 \fB\-\-test\-slack\fR
 Send a test notification to Slack
+.TP
+\fB\-\-test\-discord\fR
+Send a test notification to Discord
```

Comment:
This is not implemented, right?

Reply:
That is correct, I thought I had pulled this when I saw the generic reporter test, will reverit and push up the change. Will reply to the PR when that is done. I also see it's reporting a discord error when executing a --test-reporter, the test succeeds but it reports a discord error to the command line even though there isn't one.

Reply:
Man page reverted and added response code for no_content to remove erroneous discord repsonse error. Ready to merge.

Reply:
Added discord embedded support, after looking at numerous test messages in discord it became hard to identify individual reports.